core/src/main/java/org/mskcc/cbio/oncokb/model/BiologicalVariant.java

```diff
@@ -85,39 +86,59 @@ public void setOncogenicPmids(Set<String> oncogenicPmids) {
         this.oncogenicPmids = oncogenicPmids;
     }
 
+    @ApiModelProperty(value = "")
+    @JsonProperty("mutationEffectAbstracts")
```

Comment:
Auto-generated code by swagger will insert this annotation automatically, just want to keep the consistancy.